test/jit/test_backends.py

```diff
@@ -34,6 +36,13 @@ def to_test_backend_multi(module, method_compile_spec):
     return torch._C._jit_to_backend("test_backend", module, method_compile_spec)
 
 
+def to_test_backend_selective(module, method_compile_spec, submodules):
+    def _to_test_backend(module):
+        return to_test_backend(module, method_compile_spec)
```

Comment:
I haven't noticed that so far. This means that all submodules have have the same method_compile_spec. Right?
More accurately, this assumes that the logic to map a module to its spec is found in _to_test_backend method.
To do it nicely this would really be a dictionary - will a dictionary with scriptModule as key work? (a list in the same order of submodules can also be used, more error prone though)
Alternatively, "submodules" could be a dictionary between submodules strings and method_comile_spec dictionaries.
What do you think would be the best solution here?

Reply:
Dictionary from fully qualified submodule path to method_compile_spec could work. For example: `{'a.b.c' : spec1, 'd.e.f': spec2}` where `a.b.c` means attribute `c` of attribute `b` of attribute `a` of `module`.

This would be all up to you because you can define `_to_{backend}`, the Python function you pass to `torch._C. _jit_to_backend_selective` however you want as long as it accepts one `ScriptModule` as an argument. In this example, my implementation of `_to_test_backend` captures `method_compile_spec` from the enclosing scope and uses the same one for all lowered modules but it doesn't have to be that way.